drivers/core_demographics_report/app/views/core_demographics_report/warehouse_reports/shared/_disabilities.haml

```diff
@@ -23,7 +23,7 @@
             - HudUtility2024.disability_types.each do |id, title|
               %tr
                 %th= title
-                %td= link_to_if @report.can_see_client_details?(current_user), number_with_delimiter(report.disability_count(id)), polymorphic_path(@report.detail_path_array, report.filter.for_params.deep_merge(key: "disability_#{id}", filters: { start: report.filter.start_date, end: report.filter.end_date })), target: :_blank
+                %td= link_to_if @report.can_see_client_details?(current_user) && @report.can_view_client_disability?(current_user, id), number_with_delimiter(report.disability_count(id)), polymorphic_path(@report.detail_path_array, report.filter.for_params.deep_merge(key: "disability_#{id}", filters: { start: report.filter.start_date, end: report.filter.end_date })), target: :_blank
```

Comment:
Do we need to enforce this in the controller as well, or does enforcing it the details.rb sufficiently block it such that if you navigate to the page you won't get any data?

Reply:
This section is preventing the link from being active if the user doesn't have the correct permission. The cannot_view_details (soon to be can_view_details) on disability_detail_hash being set to false is making it so the details page will load but show no data if user doesnt have permission to view the disability data.

Reply:
As long as the detail view doesn't include any data (which was my read, and you confirmed,) we're good.  Thanks!